_includes/head.html

```diff
@@ -1,6 +1,12 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:description" content="UnetStack3 is a major milestone in the evolution of UnetStack, debuting a software-defined modem that can run on your laptop or on an embedded computer" />
```

Comment:
Is this going to be the content for each and every blog post?

Reply:
Removed the description as none of the posts currently have a description in the header.